README.md

```diff
@@ -2,6 +2,8 @@
 
 Terraform module deploying a Spacelift worker pool on AWS EC2 using an autoscaling group.
 
+This module can optionally deploy a Lambda function to autos-scale the worker pool. The function adds or removes workers depending on the worker pool queue length.
```

Comment:
I do not see the variable to make the autoscaler optional.

Reply:
I've added the parameter